store/homeSection.ts

```diff
@@ -0,0 +1,201 @@
+  },
+  async getAudioResumes({ dispatch }) {
+    try {
+      const { data } = await this.$api.items.getResumeItems({
+        userId: this.$auth.user.Id,
+        limit: 12,
+        fields: [ItemFields.PrimaryImageAspectRatio],
+        imageTypeLimit: 1,
+        enableImageTypes: [
+          ImageType.Primary,
+          ImageType.Backdrop,
+          ImageType.Thumb
+        ],
+        enableTotalRecordCount: false,
+        mediaTypes: ['Audio']
+      });
+
+      dispatch('getAudioResumesSuccess', data);
+    } catch (err) {
+      dispatch('getAudioResumesFailure', err);
+    }
+  },
+  getAudioResumesSuccess({ commit }, response: BaseItemDtoQueryResult) {
+    commit('ADD_AUDIO_RESUMES', {
+      audioResumes: response.Items
+    });
+  },
+  getAudioResumesFailure: async ({ dispatch }, error) => {
+    await dispatch(
+      'snackbar/pushSnackbarMessage',
+      {
+        message: error.message || 'Getting audio resumes issue',
+        color: 'error'
+      },
+      {
+        root: true
+      }
+    );
+  },
+  async getVideoResumes({ dispatch }) {
+    try {
+      const { data } = await this.$api.items.getResumeItems({
+        userId: this.$auth.user.Id,
+        limit: 12,
+        fields: [ItemFields.PrimaryImageAspectRatio],
+        imageTypeLimit: 1,
+        enableImageTypes: [
+          ImageType.Primary,
+          ImageType.Backdrop,
+          ImageType.Thumb
+        ],
+        enableTotalRecordCount: false,
+        mediaTypes: ['Video']
+      });
+
+      dispatch('getVideoResumesSuccess', data);
+    } catch (err) {
+      dispatch('getVideoResumesFailure', err);
+    }
+  },
+  getVideoResumesSuccess({ commit }, response: BaseItemDtoQueryResult) {
+    commit('ADD_VIDEO_RESUMES', {
+      videoResumes: response.Items
+    });
+  },
+  getVideoResumesFailure: async ({ dispatch }, error) => {
+    await dispatch(
+      'snackbar/pushSnackbarMessage',
+      {
+        message: error.message || 'Getting video resumes issue',
+        color: 'error'
+      },
+      {
+        root: true
+      }
+    );
+  },
+  async getUpNext({ dispatch }, { parentId }: { parentId: string }) {
+    try {
+      const { data } = await this.$api.tvShows.getNextUp({
+        userId: this.$auth.user.Id,
+        limit: 12,
+        fields: [ItemFields.PrimaryImageAspectRatio],
+        imageTypeLimit: 1,
+        enableImageTypes: [
+          ImageType.Primary,
+          ImageType.Backdrop,
+          ImageType.Thumb
+        ],
+        parentId
+      });
+
+      dispatch('getUpNextSuccess', data);
+    } catch (err) {
+      dispatch('getUpNextFailure', err);
+    }
+  },
+  getUpNextSuccess({ commit }, response: BaseItemDtoQueryResult) {
+    commit('ADD_UP_NEXT', {
+      upNext: response.Items
+    });
+  },
+  getUpNextFailure: async ({ dispatch }, error) => {
+    await dispatch(
+      'snackbar/pushSnackbarMessage',
+      {
+        message: error.message || 'Getting up next issue',
```

Comment:
These should probably be translated. Also, I think that " Unable to get up next items" would send better here

Reply:
This message is set in case the server forget to send an error message. It is almost impossible to reach that message, but I added it in case something went wrong and the passed error doesn't have a message. To be honest we can get rid of it.

Reply:
In that case wouldn't it be better to use the error code to get a translated error message (like we do for login). AFAIK the server won't send the messages pre-translated/

Reply:
Well if the backend uses correctly the HTTP codes, we can wrap that in the `snackBar` functionality and just send the error and the snack bar will handle the rest providing the correct translated message. 

On the other hand, I don't see this happening in this PR. 

@MrTimscampi What do you think? I can make a PR adding middleware to the store checking the error code passed to the snackbar and replace the error message with a translated one for each HTTP code.

Reply:
The middleware proposal seems good, since we seem to do these kind of checks everywhere :)

But I agree, that's a job for another PR, not for this one. I think we can get rid of the fallback message, we should never reach it, I believe (And if we do, it's time for some refactoring of the error handling, probably)

Reply:
Ok, sounds fine to me

Reply:
Done! I'll get the refactoring for the error codes. So after the tvShows PR gets in I'll start with the refactoring and then continue with the refactoring of the calls moved to vuex.